### ISSUE TITLE
Incomplete soft-dotting

### ISSUE THREAD

**Issue by dscorbett** [JetBrains/JetBrainsMono#345]
Soft dots are only removed from ⟨i⟩ and ⟨j⟩. The following should also lose their dots in the same contexts:

* U+012F LATIN SMALL LETTER I WITH OGONEK
* U+0456 CYRILLIC SMALL LETTER BYELORUSSIAN-UKRAINIAN I
* U+0458 CYRILLIC SMALL LETTER JE
* U+1ECB LATIN SMALL LETTER I WITH DOT BELOW

**Comment by philippnurullin** [comment]
Ok, will update in the next version.

**Comment by philippnurullin** [comment]
Hi @dscorbett Sorry can't find this problem in the font. Soft dotting is complete starting from v2.200. Maybe you looking at the dates source files?

**Comment by dscorbett** [comment]
Soft-dotting is not complete. Using JetBrainsMono-Regular.ttf from [version 2.200](https://github.com/JetBrains/JetBrainsMono/releases/tag/v2.200), here is <U+0069, U+0304, U+006A, U+0304, U+012F, U+0304, U+0456, U+0304, U+0458, U+0304, U+1ECB, U+0304>.
![īj̄į̄і̄ј̄ị̄](https://user-images.githubusercontent.com/1124347/98546987-84346680-2265-11eb-84ba-0337ef79803b.png)
The dots are only removed from the first two letters.

**Comment by philippnurullin** [comment]
Yes, looks like you right. And it looks like i have no idea what "soft-dotting" is. )
Do you happen to know the source where i can read about it, or you have a font as an example?

**Comment by dscorbett** [comment]
[Principle P9 in chapter 3 of <i>The Unicode Standard</i>](https://www.unicode.org/versions/Unicode13.0.0/ch03.pdf#page=43) describes soft dots.
